htdocs/compta/bank/bankentries_list.php

```diff
@@ -1559,7 +1559,12 @@
 		// Payment type
 		if (!empty($arrayfields['type']['checked'])) {
 			print '<td class="tdoverflowmax100">';
-			$labeltype = ($langs->trans("PaymentTypeShort".$objp->fk_type) != "PaymentTypeShort".$objp->fk_type) ? $langs->trans("PaymentTypeShort".$objp->fk_type) : $langs->getLabelFromKey($db, $objp->fk_type, 'c_paiement', 'code', 'libelle', '', 1);
+			if (!empty($objp->fk_type)) {
+				$labeltype=($langs->trans("PaymentTypeShort" . $objp->fk_type) != "PaymentTypeShort" . $objp->fk_type) ? $langs->trans("PaymentTypeShort" . $objp->fk_type) : $langs->getLabelFromKey($db,
+					$objp->fk_type, 'c_paiement', 'code', 'libelle', '', 1);
+			} else {
+				$labeltype="";
+			}
```

Comment:
This way your code is more easier to read
```php
$labeltype = !empty($objp->fk_type) ? 
    ($langs->trans("PaymentTypeShort" . $objp->fk_type) != "PaymentTypeShort" . $objp->fk_type ? 
        $langs->trans("PaymentTypeShort" . $objp->fk_type) : 
        $langs->getLabelFromKey($db, $objp->fk_type, 'c_paiement', 'code', 'libelle', '', 1)) : 
    "";

```

Reply:
It seems to be easier to read like that.
I will just replace "trans" with "transnoentitiesnoconv" (see commit https://github.com/Dolibarr/dolibarr/commit/0542c956dcdc582514fe3a273acbecac2ab4e7e4 on develop)